packages/@o3r/core/package.json

```diff
@@ -39,7 +39,9 @@
     "@schematics/angular": "~18.2.0",
     "chokidar": "^3.5.2",
     "globby": "^11.1.0",
+    "minimist": "^1.2.6",
```

Comment:
minimist is needed for a script that is not exposed anymore (instead we maybe can delete the following file `packages/@o3r/core/schematics/scripts/sub-entry-imports.ts`)